sbc_compassion/tools/patternrecognition.py

```diff
@@ -82,7 +82,7 @@ def patternRecognition(image, pattern, crop_area=None,
         return kp1, kp2, good
     if len(good) >= threshold:
         # put in a np.array the position of the image's keypoints
-        keypoints = np.array([kp1[i[0].trainIdx].pt for i in good])
+        keypoints = np.array([kp1[i[0].queryIdx].pt for i in good])
```

Comment:
Same as the comment above.
As I have done a switch between des1 and des2, I need to take the other Id (query is for one image and train the other one).